consumer.go

```diff
@@ -138,9 +190,9 @@ func (c *Consumer) Scan(ctx context.Context, fn ScanFunc) error {
 	return <-errc
 }
 
-// ScanShard loops over records on a specific shard, calls the callback func
+// ScanShardContinuous loops over records on a specific shard, calls the callback func
 // for each record and checkpoints the progress of scan.
-func (c *Consumer) ScanShard(ctx context.Context, shardID string, fn ScanFunc) error {
```

Comment:
I'd rather not break the public API if possible. lemme put some thought into this one

Reply:
Sounds good : )